composer.json

```
@@ -54,12 +55,12 @@
     },
     "require": {
         "php": ">=7.1.0",
-        "laravel/framework": "5.5.*|5.6.*|5.7.*|5.8.*"
+        "laravel/framework": "5.5.*|5.6.*|5.7.*|5.8.*|6.*.*"
```

Since Laravel 6 is Semver, let's drop the `*`

```suggestion
        "laravel/framework": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0"
```